main.go

```diff
@@ -28,15 +29,19 @@ var signingFileExts = []string{".mf", ".rsa", ".dsa", ".ec", ".sf"}
 // --- Models
 // -----------------------
 
-// ConfigsModel ...
-type ConfigsModel struct {
-	BuildArtifactPath  string
-	KeystoreURL        string
-	KeystorePassword   string
-	KeystoreAlias      string
-	PrivateKeyPassword string
-	JarsignerOptions   string
-	OutputName         string
+type configs struct {
+	BuildArtifactPath  string `env:"android_app,required"`
+	KeystoreURL        string `env:"keystore_url,required"`
+	KeystorePassword   string `env:"keystore_password,required"`
+	KeystoreAlias      string `env:"keystore_alias,required"`
+	PrivateKeyPassword string `env:"private_key_password"`
+	JarsignerOptions   string `env:"jarsigner_options"`
+	OutputName         string `env:"output_name"`
+
+	VerboseLog bool `env:"verbose_log,opt[yes,no]"`
+
+	// Deprecated
+	APKPath string `env:"apk_path"`
```

Comment:
Why add a deprecated value?

Reply:
The `apk_path` is deprecated.
Step.yml:
`Android App Bundle (.aab)` or `Android Aplication Package (.apk)`. Deprecated, use `android_app` instead.